README.md

```diff
@@ -108,3 +114,38 @@ When running the simulation with the tracks you can control the flippers axe wit
 `rostopic pub -1 /markhor/flipper_<fl|fr|rl|rr>_position_controller/command std_msgs/Float64 "data: <0 = 180deg -3 = -180deg>"`
 
 In the future this control will be done with a hardware interface.
+
+
+
+# Running using Docker
```

Comment:
Docker? The files don't exist in this branch, no?

Reply:
Thanks, I was due to the old pr